src/pybind/mgr/dashboard/frontend/src/app/ceph/block/pool/pool.ts

```diff
@@ -0,0 +1,59 @@
+export interface Pool {
```

Comment:
Could you remove this file? It is not being use and I think it's better not to typify ceph structures.

Reply:
Sorry this is part of the upcoming PR should mend to be in here.